hack/build-container-hostpathplugin.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+NAME=hostpathplugin
+TAG=canary
```

Comment:
Make them vars in Makefile and force users to set it and set them from the build environment for automated builds. That way, users can specify their own DOCKER_PREFIX & DOCKER_TAG.

Reply:
Let me see what I can do. This is kind of complicated with so many drivers under the same tree. I wish each was its own repo.